examples/caching_nodes/run.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import sys
+
+from hamilton import driver
+
+import data_loaders
+import business_logic
+
+logging.basicConfig(stream=sys.stdout)
+
+# This is empty, we get the data from the data_loaders module
+initial_columns = {}
+
+dr = driver.Driver(initial_columns, business_logic, data_loaders)
```

Comment:
This should have the adapter passed in here right?

Reply:
Correct, fixed now.